setup.cfg

```diff
@@ -56,8 +56,10 @@ install_requires =
     numpy==1.19.3; python_version=='3.9' and platform_machine not in 'arm64|loongarch64' and platform_python_implementation != 'PyPy'
     # Note that 1.21.3 was the first version with a complete set of 3.10 wheels,
     # however macOS was broken and it's safe to build against 1.21.6 on all platforms (see gh-28 and gh-45)
-    numpy==1.21.6; python_version=='3.10' and platform_machine!='loongarch64' and platform_python_implementation != 'PyPy'
+    numpy==1.23.2; python_version=='3.10' and platform_machine!='loongarch64' and platform_python_implementation != 'PyPy'
```

Comment:
Grr, this change was unintended, reverting that

Reply:
Force-pushed commit 0688c4bbf25322194e249ec1f8072961f8fe02e3 to repair this. Done within minutes, so should be okay.